ui/app/templates/components/license-banners.hbs

```diff
@@ -1,45 +1,54 @@
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @onDismiss={{fn this.dismissBanner "expired"}}
+      data-test-license-banner-expired
+      as |A|
     >
-      <DocLink @path="/vault/tutorials/enterprise/hashicorp-enterprise-license">
-        <Icon @name="learn-link" />
-        Read documentation
-      </DocLink>
-      <button type="button" class="close-button" {{on "click" (fn this.dismissBanner "expired")}} data-test-dismiss-expired>
-        <Icon @name="x" aria-label="dismiss license expired warning" />
-      </button>
-    </AlertBanner>
+      <A.Title>License expired</A.Title>
+      <A.Description>
+        Your Vault license expired on
+        {{date-format @expiry "MMM d, yyyy"}}. Add a new license to your configuration and restart Vault.
+      </A.Description>
+      <A.Description class="has-top-margin-xs">
+        <DocLink @path="/vault/tutorials/enterprise/hashicorp-enterprise-license">
+          <Icon @name="learn-link" />
+          Read documentation
+        </DocLink>
+      </A.Description>
+    </Hds::Alert>
   </div>
 {{else if (and (lte this.licenseExpiringInDays 30) (not this.warningDismissed))}}
-  <div class="license-banner-wrapper" data-test-license-banner data-test-license-banner-warning>
-    <AlertBanner
-      @type="warning"
-      @title="Vault license expiring"
-      @message="Your Vault license will expire in {{this.licenseExpiringInDays}} days at {{date-format
-        @expiry
-        'hh:mm:ss a'
-      }} on {{date-format @expiry 'MMM d, yyyy'}}. {{if
-        @autoloaded
-        'Add a new license to your configuration.'
-        'Keep in mind that your next license will need to be autoloaded'
-      }}"
-      class="message-marginless"
+  <div class="license-banner-wrapper">
+    <Hds::Alert
+      @type="inline"
+      @color="warning"
+      @onDismiss={{fn this.dismissBanner "warning"}}
+      data-test-license-banner-warning
+      as |A|
     >
-      <DocLink @path="/vault/tutorials/enterprise/hashicorp-enterprise-license">
-        <Icon @name="learn-link" />
-        Read documentation
-      </DocLink>
-      <button type="button" class="close-button" {{on "click" (fn this.dismissBanner "warning")}} data-test-dismiss-warning>
-        <Icon @name="x" aria-label="dismiss license expire soon warning" />
-      </button>
-    </AlertBanner>
+      <A.Title>Vault license expiring</A.Title>
+      <A.Description>
+        Your Vault license will expire in
+        {{this.licenseExpiringInDays}}
+        days at
+        {{date-format @expiry "hh:mm:ss a"}}
+        on
+        {{date-format @expiry "MMM d, yyyy"}}.
+        {{if
+          @autoloaded
+          "Add a new license to your configuration."
+          "Keep in mind that your next license will need to be autoloaded"
+        }}
+      </A.Description>
+      <A.Description class="has-top-margin-xs">
+        <DocLink @path="/vault/tutorials/enterprise/hashicorp-enterprise-license">
```

Comment:
Not using the HDS [standalone](https://helios.hashicorp.design/components/link/standalone) link with the goal being to use that component within <DocLink> itself